storage/statedb/database.go

```diff
@@ -1153,6 +1153,21 @@ func (db *Database) SaveTrieNodeCacheToFile(filePath string) error {
 	return nil
 }
 
+// DumpPeriodically atomically saves fast cache data to the given dir with the specified interval.
+func (db *Database) SaveCachePeriodically(c *TrieNodeCacheConfig, stopCh <-chan struct{}) {
+	ticker := time.NewTicker(c.FastCacheSavePeriod)
+	defer ticker.Stop()
+
+	for {
+		select {
+		case <-ticker.C:
+			db.SaveTrieNodeCacheToFile(c.FastCacheFileDir)
```

Comment:
@ehnuje If the period is too short, this can call `SaveTrieNodeCacheToFile` again during saving.
So could you remove goroutine in `SaveTrieNodeCacheToFile` and change caller code of the `SaveTrieNodeCacheToDisk` API?

Reply:
Answered in the below comment, please check.